mantidproject/mslice
# 555 add workspace wrapper

Added the command AddWorkspaceToDisplay(workspace, workspace_name) to the command line interface for MSlice. This wraps the commands for adding a workspace to the workspace list in the GUI into one command.

**To test:**
In the MSlice command interface, use the new command AddWorkspaceToDisplay(workspace, workspace_name) to add a workspace to the GUI of MSlice. Check the workspace list in the GUI of MSlice whether the new workspace is included. 

An easy test can be done with the following code:

from mantid.simpleapi import *
import matplotlib.pyplot as plt
import numpy as np
import mslice.cli as mc

from mslice.workspace.histogram_workspace import HistogramWorkspace

signal = list(range(0, 100))
error = np.zeros(100) + 2
hs_ws = HistogramWorkspace(CreateMDHistoWorkspace(Dimensionality=2, Extents='0,100,0,100', SignalInput=signal, ErrorInput=error, NumberOfBins='10,10', Names='Dim1,Dim2', Units='U,U', OutputWorkspace='testHistoWorkspace',), 'testHistoWorkspace')

mc.AddWorkspaceToDisplay(hs_ws, hs_ws.name)


Fixes #[555](https://github.com/mantidproject/mslice/issues/555).
